script.js

```diff
@@ -0,0 +1,65 @@
+console.log("test js on");
+// first task
+const dishes = [
+  "Spaghetti",
+  "Lasagne",
+  "Beans on toast",
+  "Porridge",
+  "Tomato soup",
+  "Salmon",
+];
+for (let i = 0; i < dishes.length; i++) {
+  console.log(dishes[i]); // dirty but ...
+}
+/* // 2nd task - Using JavaScript, create a button and add it to the html (the document)
+When the button is clicked:
+- Insert an h1 tag with the text “Javascript test”
+- Change the background color of the body */
+
+const body = document.body;
+const button = document.createElement("button");
+const tag = document.createElement("h1");
+function addTagChangeBg() {
+  tag.innerText = `Javascript test`;
+  body.append(tag);
+  body.style = `background-color: #CCC;`;
+  // function for click-event that adds h1 and changes background to gray
+}
+button.innerText = `BUTTON THAT DOES THINGS`;
+body.append(button);
+
+button.addEventListener("click", addTagChangeBg);
+// Task 3
+/* 
+Make an API call using the Fetch API. Make use of the following API:
+https://reqres.in/api/users
+Display the first_name of the first three users in the DOM */
+
+fetch("https://reqres.in/api/users")
+  .then((response) => response.json())
+  .then((data) => {
+    for (let i = 0; i < 3; i++) {
+      const span = document.createElement("span");
+      span.innerText = `${data[i]?.first_name} `; // can't target sorry. This is what i need to learn from the test - bloody promises :)
```

Comment:
Please keep the comments professional! A better comment would have been: Thanks to this test, I found out that I need to improve my knowledge about APIs. for example. If I was a recruiter looking in your Github repositories, that would have gived me a better impression
Going back to the issue, You have done well overall, you just needed to find the target wich was in `data.data[i]?.first_name`.
The other tip is that, you could use `ul` and `li` to display the names, instead of a span.

Reply:
Will do. I am mostly leaving these fun (read cringe) comments to remember the points i really need remembering / improving. Things i can solve with no frustration usually are on a bit more professional level. Thanks for the feedback!